api/api_full.go

```diff
@@ -850,6 +863,10 @@ type ChannelAvailableFunds struct {
 	ConfirmedAmt types.BigInt
 	// PendingAmt is the amount of funds that are pending confirmation on-chain
 	PendingAmt types.BigInt
+	// AvailableAmt is part of ConfirmedAmt that is available for use (pre-allocated)
+	AvailableAmt types.BigInt
```

Comment:
I suggest re-wording these comments to emphasize the difference between reserved and non-reserved funds. It may be worth using some longer names for the variables themselves to make that clear